### PR TITLE
[IOTDB-1234] fix ImportCsv tool cannot parse empty string

### PR DESCRIPTION
**Pull request by HelKim**

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java [hunk 1]
```diff
@@ -47,29 +47,9 @@
 import org.apache.iotdb.db.qp.executor.PlanExecutor;
 import org.apache.iotdb.db.qp.logical.Operator.OperatorType;
 import org.apache.iotdb.db.qp.physical.PhysicalPlan;
-import org.apache.iotdb.db.qp.physical.crud.AggregationPlan;
-import org.apache.iotdb.db.qp.physical.crud.AlignByDevicePlan;
+import org.apache.iotdb.db.qp.physical.crud.*;
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java:50]: general,  do not use import  *...

Comment (HelKim) [reply server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java]: > general, do not use import *...

Thanks

**Comment by jixuan1989** [comment]
thanks for contribution!

**Comment by jixuan1989** [comment]
Hi, though the PR is merged, it left one thing... 
You'd better add a Test for the case.

**Pull request merged**